src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -2131,20 +2146,12 @@ describe('SavedObjectsRepository', () => {
           ],
         },
       });
-      const results = await savedObjectsRepository.find({ type: ['hiddenType', 'config'] });
-      expect(results).toEqual({
-        total: 1,
-        saved_objects: [
-          {
-            id: 'one',
-            references: [],
-            type: 'config',
-            updated_at: mockTimestamp,
-          },
-        ],
-        page: 1,
-        per_page: 20,
-      });
+      try {
+        await savedObjectsRepository.find({ type: ['hiddenType', 'config'] });
+        throw new Error('expected find() to reject');
```

Comment:
nit: you can also use expect().rejects to assert a promise rejects like:
`expect(savedObjectsRepository.find(...)).rejects.toThrowErrorMatchingInlineSnapshot();`